feature/platform/tests/optics_power_and_bias_current_test/README.md

```diff
@@ -1,49 +1,102 @@
+samples each optics part number, connect both two optical ethernet interfaces
+to Automatic Test Equipment (ATE). 
+
 
 ## Procedure
 
 *   Connect at least one optical ethernet interface to ATE.
-*   Verify that the following optics telemetry paths exist for the installed
-    optics.
-    *   /components/component/transceiver/physical-channels/channel/state/input-power/instant
-    *   /components/component/transceiver/physical-channels/channel/state/output-power/instant
-    *   /components/component/transceiver/physical-channels/channel/state/laser-bias-current/instant
-*   Verify the optics power is updated after the interface is flapped.
+*   Step 1: Using components/component/[name=%s]/state get the list of transceivers and validate
+    following leafs are set:
+
+    *   /components/component/[name=%s]/state/mfg-name
+    *   /components/component/[name=%s]/transceiver/state/form-factor
+    *   /components/component/[name=%s]/state/serial-no
+    *   /components/component/[name=%s]/state/part-no
+    *   /components/component/[name=%s]/state/firmware-version
+
+    *   Using /interfaces/interface[name=%s]/state get the list of Interfaces and
+        validate following leafs are set:
 
-    *   Enable an interface on the DUT.
-    *   Verify the optics input and output power are in the normal range.
-    *   Disable or shut down the interface on the DUT.
-    *   Verify the optics output power is updated to very low value.
-    *   Re-enable the interface on the DUT
-    *   Verify the optics output power is updated to the value in the normal
-        range again
+        *   /interfaces/interface[name=%s]/state/physical-channel
+        *   /interfaces/interface[name=%s]/state/transceiver
 
-*   Using components/component[name=%s]/state get the list of Transceivers and
-    validate following leafs are set:
+    *   Verify that the list of transceivers received are the same. This is only a
+        consistency check that the vendor implemented the model correctly.
 
-    *   /components/component[name=%s]/state/mfg-name
-    *   /components/component[name=%s]/transceiver/state/form-factor
+        *   /components/component[name=%s]/state
+        *   /interfaces/interface[name=%s]/state/transceiver
 
-*   Using /interfaces/interface[name=%s]/state get the list of Interfaces and
-    validate following leafs are set:
+*   Step 2: Get list of components of type TRANSCEIVER. Verify the instant value is
+    between the corresponding lower and upper thresholds for both
+    [severity]=WARNING and [severity]=CRITICAL:
+    *   Module case temperature
+        *   /components/component/transceiver/thresholds/threshold/state/module-temperature-lower
+        *   /components/component/transceiver/thresholds/threshold/state/module-temperature-upper
+        *   /components/component/transceiver/thresholds/threshold/state/severity
+        *   /components/component/state/temperature/instant
+    *   Tx output power
```

Comment:
Maybe we can call out that every physical channel is expected? For example if there are four physical channels in the transceiver we must see all of them.

This also applies to RX below.

Reply:
Updated to call out multiple lanes.